integration/tests/tracer/test_tracer_debug_methods.py

```diff
@@ -505,10 +506,13 @@ def test_debug_get_modified_accounts_by_invalid_hash(self, params):
 
     def test_debug_get_raw_transaction(self):
         sender_account = self.accounts[0]
-        nonce = self.web3_client.get_nonce(sender_account.address)
-        transaction = self.web3_client.make_raw_tx(sender_account, nonce=nonce, amount=0.1)
+        transaction = self.web3_client.make_raw_tx(sender_account)
+        transaction["data"] = GOOD_CALLDATA[0]
+        transaction["chainId"] = self.web3_client.eth.chain_id
+        transaction["gas"] = self.web3_client.eth.estimate_gas(transaction)
```

Comment:
chainId and gas were added by make_raw_tx(
and there are data parametr too 
transaction = self.web3_client.make_raw_tx(sender_account, data=GOOD_CALLDATA[0])

Reply:
agree and fixed